.pre-commit-config.yaml

```diff
@@ -0,0 +1,8 @@
+---
+
+repos:
+  - repo: https://github.com/psf/black
+    rev: 20.8b1
+    hooks:
+      - id: black
+        args: [--line-length=88]
```

Comment:
```suggestion
        args: [--line-length=100]
```

Reply:
I think that's what we're running in cg.

Reply:
It will clash with the actions black which uses the defaults of 88. I do not know how to add options in the actions. Could we do that in a later PR?

Reply:
Sure!